src/main/java/AddCommand.java

```diff
@@ -0,0 +1,20 @@
+public class AddCommand extends Command{
```

Comment:
Perhaps you could add JavaDoc comments for classes? I noticed some other classes were missing JavaDoc comments as well.